src/main/java/ogc/rs/database/DatabaseServiceImpl.java

```diff
@@ -848,8 +848,8 @@ public Future<JsonObject> postStacCollection(JsonObject jsonObject) {
         JsonArray temporalJsonArray = jsonObject.getJsonObject("extent").getJsonObject("temporal").getJsonArray("interval").getJsonArray(0);
         String[] temporalArray = temporalJsonArray.stream().map(Object::toString).toArray(String[]::new);
 
-        LOGGER.debug("id: {}, title: {}, description: {}, datetimeKey: {}, crs: {}, bboxArray: {}, temporalArray: {}, license: {}",
-                id, title, description, datetimeKey, crs, bboxArray, temporalArray, license);
+        LOGGER.debug("id: {}, title: {}, description: {}, datetime: {}, crs: {}, bboxArray: {}, temporalArray: {}, license: {}",
```

Comment:
Again, it is going to be `datetimeKey` that has to be manually set to `"datetime"`.